src/components/venuePortal.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
```

Comment:
I really like the work youve done on the venue portal, so much nicer to have them pulled out of app.js body. This all looks super clean and readable, you have short succint methods  boom